ddht/v5_1/network.py

```diff
@@ -143,7 +171,7 @@ async def talk(
     ) -> bytes:
         if endpoint is None:
             endpoint = self._endpoint_for_node_id(node_id)
-        response = await self.client.talk(endpoint, node_id, protocol, payload)
+        response = await self.client.talk(endpoint, node_id, protocol, payload,)
```

Comment:
This one is a bit strange. Why are these kinds of updates happening in your recent PRs, did you update your version of a linter?

Reply:
Likely happened because it was originally short and fit on the line.  Then I added something that made `black` wrap the line.  Then I visually saw that it was missing a dangly comma and maybe added it manually.  Then I removed whatever extra stuff made the line have to be wrapped and `black` unwrapped it, but it didn't drop the dangly comma....